cloud-consumption/src/test/java/com/sequenceiq/consumption/service/ConsumptionServiceTest.java

```diff
@@ -144,4 +159,149 @@ void testCreateWhenConsumptionAlreadyExists() {
         verify(consumptionDtoConverter, Mockito.times(0)).creationDtoToConsumption(eq(consumptionCreationDto));
     }
 
+    @Test
+    public void testFindAllConsumption() {
```

Comment:
Possibly to be renamed if `underTest.findAllConsumption()` gets renamed. :smile: